tsc-responsibilities.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: TSC responsibilities
+nav_order: 21
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# The Hyperledger Technical Steering Committee (TSC) responsibilities
+
+The [Hyperledger Charter](https://www.hyperledger.org/about/charter) states that the TSC is responsible for:
+
+1. coordinating the technical direction of the Hyperledger Foundation;
+
+2. approving project proposals (including, but not limited to, incubation, deprecation and changes to a project’s charter or scope) in accordance with a project lifecycle document to be developed, approved and maintained by the TSC;
```

Comment:
does it also includes labs?

Reply:
No. Labs have been designed to provide a space that's explicitly not under direct governance of the TSC and do not imply endorsement from it.